stable/traefik/templates/daemonset.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if eq .Values.deployType "Daemonset" }}
```

Comment:
Shouldn't Daemonset be [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)? We should keep consistency with kubernetes' resources names.

Reply:
ok